src/components/Guides/PlaylistPage.tsx

```diff
@@ -1,35 +1,115 @@
 import React from 'react';
+import { Typography } from '@material-ui/core';
+import { withStyles, Theme } from '@material-ui/core/styles';
+import { styles } from './PlaylistStyles';
 import { useSelector } from 'react-redux';
 import { RootState } from '../../store/reducers';
 import StandardCard from '../Cards/StandardCard';
+import { ReactComponent as InstructionIcon } from './InstructionIcon.svg';
+import { ReactComponent as VideoIcon } from './VideoIcon.svg';
 
 type PlaylistPageProps = {
-  lessonId: string,
-}
+  lessonId: string;
+  classes: any;
+};
 
-const PlaylistPage = ({ lessonId }: PlaylistPageProps) => {
+const PlaylistPage = ({ lessonId, classes }: PlaylistPageProps) => {
   const lessons = useSelector((state: RootState) => state.lessons);
   const resources = useSelector((state: RootState) => state.resources);
+
   const currentLesson = lessons[lessonId];
+  const countMedia = (media: string) =>
```

Comment:
I remember you mentioning talking about setting up some more utils files! we don't have to resolve this necessarily in this PR but we could see if we can add that